VisualPinball.Unity/VisualPinball.Unity/VPT/Ball/BallData.cs

```diff
@@ -26,7 +26,43 @@ internal struct BallData : IComponentData
 		public float3 Position;
 		public float3 EventPosition; // m_lastEventPos
 		public float3 Velocity;
+
+		/// <summary>
+		/// AngularVelocity  -  german: Winkelgeschwindigkeit
+
+		///		* Set to 0 at Manual Roll
+		///			(in BallManualRoll(in Entity entity, in float3 targetWorldPosition)
+		///			(which is not used anywhere)

```

Comment:
It's used [here](https://github.com/ravarcade/VisualPinball.Engine.Unity.ImgGUI/blob/master/Monitors/BallMonitor.cs#L36), but we can probably remove it unless we want to make the physics API agnostic again.

Reply:
Oh, OK. did not see this and it was not signed as a "reference" inside of Visual Studio. Will add that. I wondered why it was not used. Now I know that I have to search for the Function name across the repo. Thanks for pointing out, will add that.

Reply:
Oh, now i see, it's a Plugin - a physics debugger... had i known about this... Seems nice